### ISSUE TITLE
Update to latest Zeebe Client

### ISSUE THREAD

**Issue by salaboy** [zeebe-io/zeebe-cloud-events-router#3]
Update to Latest version of Spring Zeebe

**Comment by gaurav9822** [comment]
@salaboy can I take this one?

**Comment by salaboy** [comment]
@gaurav9822 yes of course.. give it a go and see if you can add some tests to verify that the new client is working.

**Comment by gaurav9822** [comment]
@salaboy 0.24.0 is the latest version for "zeebe.version"

**Comment by salaboy** [comment]
@gaurav9822 you are right.. 0.25 is coming along but it is still not published.. do you want to start by adding tests? I can rename the issue if you want

**Comment by gaurav9822** [comment]
@salaboy I can work on adding tests but can you specify tests for which class/behaviour?

**Comment by salaboy** [comment]
@gaurav9822 sure.. 
I would suggest starting with a set of tests for this endpoint: 

https://github.com/zeebe-io/zeebe-cloud-events-router/blob/master/src/main/java/io/zeebe/cloud/events/router/ZeebeCloudEventsRouterController.java#L229

Which contains the core functionality of the router. 

Then test the Worker code which is here:

https://github.com/zeebe-io/zeebe-cloud-events-router/blob/master/src/main/java/io/zeebe/cloud/events/router/ZeebeCloudEventsRouterWorker.java#L28

If we manage to add these two tests in a single PR that will be awesome!
